File: PolicyAnalyzer/GPLookup/PolSettingChoice_t.cs
using System;

namespace GPLookup
{
	// Token: 0x02000032 RID: 50
	public class PolSettingChoice_t
	{
		// Token: 0x06000087 RID: 135 RVA: 0x00002B30 File Offset: 0x00000D30
		public PolSettingChoice_t(PolLocation_t polLoc, string sValue)
		{
			this.m_PolLoc = polLoc;
			this.m_sSettingChoice = sValue;
		}

		// Token: 0x17000004 RID: 4
		// (get) Token: 0x06000088 RID: 136 RVA: 0x00002B46 File Offset: 0x00000D46
		public PolLocation_t PolLocation
		{
			get
			{
				return this.m_PolLoc;
			}
		}

		// Token: 0x17000005 RID: 5
		// (get) Token: 0x06000089 RID: 137 RVA: 0x00002B4E File Offset: 0x00000D4E
		public string SettingChoice
		{
			get
			{
				return this.m_sSettingChoice;
			}
		}

		// Token: 0x04000125 RID: 293
		private PolLocation_t m_PolLoc;

		// Token: 0x04000126 RID: 294
		private string m_sSettingChoice;
	}
}

File: Python/gplookup/pol_setting_choice_t.py
# gplookup/pol_setting_choice_t.py
# Exact full port of PolSettingChoice_t.cs

from .pol_location_t import PolLocation_t

class PolSettingChoice_t:
    def __init__(self, pol_loc: PolLocation_t, s_value: str):
        self.m_PolLoc = pol_loc
        self.m_sSettingChoice = s_value

    @property
    def PolLocation(self):
        return self.m_PolLoc

    @property
    def SettingChoice(self):
        return self.m_sSettingChoice